internal/mode/static/manager.go

```diff
@@ -147,7 +147,7 @@ func StartManager(cfg config.Config) error {
 
 	if cfg.MetricsConfig.Enabled {
 		constLabels := map[string]string{"class": cfg.GatewayClassName}
-		ngxCollector, err := collectors.NewNginxMetricsCollector(constLabels)
+		ngxCollector, err := collectors.NewNginxMetricsCollector(constLabels, cfg.PlusEnabled)
```

Comment:
I'm not a huge fan of passing booleans to functions. It's typically an indication that the function has two different responsibilities. 

Can we add another function for creating a plus metrics collector and check the boolean here?